format/FlightSql.proto

```diff
@@ -761,6 +798,20 @@ enum SqlInfo {
   SQL_STORED_FUNCTIONS_USING_CALL_SYNTAX_SUPPORTED = 576;
 }
 
+// The level of support for Flight SQL transaction RPCs.
+enum SqlSupportedTransaction {
+  // Unknown/not indicated
+  SQL_SUPPORTED_TRANSACTION_UNKNOWN = 0;
+  // No support
+  SQL_SUPPORTED_TRANSACTION_NONE = 1;
```

Comment:
Not really - I guess in that case let's just fold them together since there's no point. (I suppose normally in protobuf you'd distinguish the two, but that doesn't apply here.)

Reply:
yea, I agree that in protobuf you'd distinguish the two in many cases, but my typical litmus test is if a consumer of the protobuf would treat the "unknown" case differently than the "none" case. And if there isn't any functional difference, it's not necessary to distinguish them.

Reply:
Yup - I ended up folding them together